src/Features/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.Executor.cs

```diff
@@ -467,15 +495,21 @@ private void UpdateAnalyzerTelemetryData(ImmutableDictionary<DiagnosticAnalyzer,
             }
         }
 
-        internal bool FullAnalysisEnabled(Project project, bool forceAnalyzerRun)
+        internal bool FullAnalysisEnabled(
+            Project project,
+            bool forceAnalyzerRun,
+            out bool compilerFullAnalysisEnabled,
+            out bool analyzersFullAnalysisEnabled)
```

Comment:
doc legal output values of this.  for example can the functino return true, but then return false for both compilerFullAnalysisEnabled and analyzersFullAnalysisEnabled?

Reply:
Sure, let me add a good doc comment with all value combinations.